gapic-generator/templates/default/service/client/_self_configure.erb

```diff
@@ -1,4 +1,29 @@
 <%- assert_locals service -%>
-@configure ||= <%= service.client_name %>::Configuration.new
+<%- method_service ||= service  -%>
+@configure ||= begin
+  default_config = <%= service.client_name %>::Configuration.new
+  <%- if service.grpc_service_config && !service.grpc_service_config.empty? -%>
+
+    <%- if service.grpc_service_config.timeout_seconds %>
```

Comment:
Ending `-%>` rather than `%>`. Same in a number of cases below, usually wherever you're not outputting with `<%=`.